src/tests/scripts/factoties.js

```diff
@@ -0,0 +1,25 @@
+import db from '../../models';
```

Comment:
Did you mean to name this file `factories`?

Reply:
Yeah, yeah let me fix it